yarn.lock

```diff
@@ -4446,9 +4446,9 @@ typedarray-to-buffer@^3.1.5:
     is-typedarray "^1.0.0"
 
 typescript@^4.1.3:
-  version "4.1.3"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.1.3.tgz#519d582bd94cba0cf8934c7d8e8467e473f53bb7"
-  integrity sha512-B3ZIOf1IKeH2ixgHhj6la6xdwR9QrLC5d1VKeCSY4tvkqhF2eqd9O7txNlS0PO3GrBAFIdr3L1ndNwteUbZLYg==
+  version "4.3.5"
```

Comment:
The `yarn.lock` file should be regenerated since you restored Typescript to `4.1.3`.

Reply:
I've already re-generated it. With the semver rule, it has upgraded to 4.3.5. I've left it because there are no changes in the built files due to this upgrade.

Reply:
👍